bitnami/fluentd/templates/NOTES.txt

```diff
@@ -3,13 +3,20 @@
   To verify that Fluentd has started, run:
 
     kubectl get all -l "app.kubernetes.io/name={{ include "fluentd.name" . }},app.kubernetes.io/instance={{ .Release.Name }}"
-{{- if and .Values.aggregator.enabled (not .Values.aggregator.configMap) -}}
+{{ if and .Values.aggregator.enabled (not .Values.aggregator.configMap) }}
   Logs are captured on each node by the forwarder pods and then sent to the aggregator pods. By default, the aggregator pods send the logs to the standard output.
   You can see all the logs by running this command:
 
     kubectl logs -l "app.kubernetes.io/component=aggregator"
   
   You can mount your own configuration files to the aggregators and the forwarders. For example, this is useful if you want to forward the aggregated logs to Elasticsearch or another service.
+{{- else }}
```

Comment:
I would change this for:

`{{- else if and (not .Values.aggregator.enabled) (not .Values.forwarder.configMap) }}`

As if the user is loading their own configuration, the forwarders may be doing something different.